### PR TITLE
added new .less file for styles for login cmp

### PR DESCRIPTION
**Pull request by lulzimfazlija**

### CODE REVIEW

File: apps/app-article/src/app-article.less [hunk 1]
```diff
@@ -0,0 +1,4 @@
+@import (reference) 'ksf-colors';
```
Comment (f-f) [inline apps/app-article/src/app-article.less]: This file should be in `less` and not in `apps/app-article/src`

File: apps/app-article/src/index.html [hunk 2]
```diff
@@ -8,6 +8,7 @@
       content="width=device-width, initial-scale=1, shrink-to-fit=no"
     />
     <meta name="theme-color" content="#000000" />
+    <link rel="stylesheet" type="text/css" href="app-article.less" />
```
Comment (f-f) [inline apps/app-article/src/index.html]:
```suggestion
    <link rel="stylesheet" type="text/css" href="../../less/app-article.less" />
```

Comment (lulzimfazlija) [reply apps/app-article/src/index.html]: changed according to your last comment :)

File: apps/app-article/src/index.html [hunk 3]
```diff
@@ -8,6 +8,7 @@
       content="width=device-width, initial-scale=1, shrink-to-fit=no"
     />
     <meta name="theme-color" content="#000000" />
+    <link rel="stylesheet" type="text/css" href="../../less/app-article.less" />
```
Comment (f-f) [inline apps/app-article/src/index.html]:
```suggestion
    <link rel="stylesheet" type="text/css" href="../../../less/app-article.less" />
```

**Pull request merged**